voluncheer/templates/chatroom/chat_homepage.html

```diff
@@ -0,0 +1,41 @@
+<!-- placeholder chat_homepage file -->
+
+{% load static %}
+
+<!DOCTYPE html>
```

Comment:
This should extend the base html template. See the profile page for an example.

Reply:
I tried it and it interferes with the demo interface.

Reply:
That is disappointing, what kind of issues are you experiencing? I can try it myself but not until later this week.

Reply:
The reason being that I copied the html file from an app online. So it doesn't currently integrate with our other UI design. I think Eden will figure it out afterward.

Reply:
In other words, this will be done but just not in this PR or story.

Reply:
I'll handle that, no need to worry